opus/application/apps/tools/file_utils.py

```diff
@@ -237,3 +237,24 @@ def get_pds_preview_images(opus_id_list, preview_jsons, sizes=None,
         image_list.append(data)
 
     return image_list
+
+def get_displayed_browse_products(opus_id, version_name='Current'):
+    """Given an opus_ids, return a list of browse products urls displayed
```

Comment:
"Given an opus_id, return a list of browse product URLs to display in the detail tab."